pages/components/modal/modal.css

```diff
@@ -0,0 +1,188 @@
+.wrapper {
+    inline-size: 85%;
+    margin-block: 0;
+    margin-inline: auto;
+}
+
+.wrapper-2 {
+    inline-size: 90%;
+    margin-block: 0;
+    margin-inline: auto;
+}
+
+.md-size {
+    font-size: 40px !important;
+    color: grey !important;
+}
+
+.dialog--hide {
+    display: none;
+}
+
+.dialog {
+    position: fixed;
+    inset-block-start: 30%;
+    inset-inline-start: 50%;
+    transform: translateX(-50%);
+    inline-size: 250px;
+    block-size: max-content;
+    background-color: var(--surface2);
+    box-shadow: 0 0 10px 0px rgba(0, 0, 0, 0.256);
+    z-index: 12;
+}
+
+.dialog__content {
+    position: relative;
+    padding-block: 1rem;
+    padding-inline: 0;
+}
+
+.dialog__clear {
+    position: absolute;
+    inset-block-start: 5px;
+    inset-inline-end: 5px;
+    opacity: .5;
+    font-size: 18px !important;
+    cursor: pointer;
+}
+
+.dialog__clear:hover {
+    opacity: 1;
+}
+
+.dialog--demo {
+    position: static;
+    margin: 0;
+    transform: translate(0);
+}
+
+.dialog--demo .dialog__clear {
+    pointer-events: none;
+}
+
+.dialog__info {
+    margin-block-end: 1.7rem;
+}
+
+.dialog__header {
+    margin-block-end: .7rem;
+    font-size: 1.5rem;
+}
+
+.dialog__description {
+    font-size: 1rem;
+    opacity: .8;
+}
+
+.dialog__cta {
+    display: flex;
+    align-items: center;
+    justify-content: end;
+    gap: 10px;
+}
+
+.dialog__action-btn {
+    border: none;
+    background: none;
+    font-size: .9rem;
+    color: var(--brand);
+    padding-inline: .4rem;
+    padding-block: .6rem;
+    border-radius: 5px;
+    text-transform: uppercase;
+    transition: .5s;
+    font-weight: bold;
+}
+
+.dialog__action-btn:hover {
+    --light-clr: #5d768c27;
+    background-color: var(--light-clr);
+}
+
+.dialog__item {
+    display: flex;
+    align-items: center;
+    gap: 20px;
+    inline-size: 100%;
+    padding-block: .5rem;
+    padding-inline: .3rem;
+}
+
+.dialog__item:hover {
+    --light-clr: #5d768c27;
+    background-color: var(--light-clr);
+    border-radius: 5px;
+}
+
+.dialog__header--confirm {
+    padding-block: .6rem;
+    padding-inline: 1rem;
+}
+
+.dialog__cta--confirm {
+    padding-block: 0;
+    padding-inline: 1rem;
+}
+
+.dialog__info--confirm {
+    --light-clr: #5d768c2f;
```

Comment:
try to define variable in the root

Reply:
It was in the root. I'm overriding it for that parent block.